OxfordBodleian/Juel-Jensen/BDLaethb5.xml

```diff
@@ -214,65 +9135,196 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <ab type="pricking">Primary pricks are visible.</ab>
+                                    <ab type="pricking">Ruling pricks are mostly visible.</ab>
+                                    <ab type="ruling" subtype="pattern">
+                                        <locus from="2ra" to="214vc"/>
+                                        Ruling pattern: 1A-1A-1A1A-1A1A/0-0/0-0/C.
+                                    </ab>
+                                    <ab type="ruling">The upper line is written above the
+                                        ruling.</ab>
+                                    <ab type="ruling">The bottom line is written above the
+                                        ruling.</ab>
                                 </layout>
                             </layoutDesc>
-                            
+
                         </objectDesc>
-                        
+
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-                           
+                                <seg type="ink">Black, red.</seg>
+                                <date notBefore="1711" notAfter="1716"/>
+                                <persName ref="PRS13094GabraKr" role="scribe"/> is mentioned as scribe in supplications throughout the 
+                                manuscript.
+                                <persName ref="PRS13115SelaKr"/> is mentioned as scribe in 
+                                supplications on <locus target="#102rc #106vc"/>. It cannot be excluded that more than one scribe were involved in the copying, 
+                                but no clear discontinuities in hand can be observed.
+                                <desc>Written by a trained hand, but slightly irregular. The characters are also spaced
+                                    irregularly.
+                                </desc>
+                                <seg
+                                    type="rubrication">Incipits (alternating red and black lines at
+                                    the top and bottom of the page for months, two continuous lines
+                                    for days, one to two continuous lines for commemorations),
+                                    elements of punctuation signs, <foreign xml:lang="gez"
+                                        >ሰላም፡</foreign>, the full names of persons involved in the
+                                    production of the manuscript (<locus target="#33ra #78vc"/>),
+                                    or, more often, only the holy names <foreign xml:lang="gez"
+                                        >ማርያም፡</foreign> and <foreign xml:lang="gez"
+                                        >ክርስቶስ፡</foreign> which form part of their name, paratextual
+                                    elements (names of the months, names of the scribes assigned to
```

Comment:
here you assume the existence of more than one scribe and the assigmnet of different quires for copying, I would expect also this idea under hands (here in fact you consider only one hand but two names); 
One possibility could be that the scribes divided the task of copying the whole Synaxarion into two codexes (not quires): one wrote the first six months, the other one the second part (which is not part of our collection. This could explain why two scribes are mentioned but only one hand is discernable in our Ms